content/posts/2025-02-26-whats_new_1_8/index.md

```diff
@@ -0,0 +1,319 @@
+
+In summary, use `cidr` for the client side address and `peerCIDR` for the server side address.
+
+#### Sampling rate per rule
+
+Each rule can have its own sampling rate.  For example, you might want the eBPF Agent to sample all external traffic on source and destination, but for internal traffic, it's sufficient to sample at 50.  Listing 5 shows how this can be done, assuming the default IP settings of 10.128.0.0/14 for pods and 172.30.0.0/16 for services.
+
+```
+spec:
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        enable: true
+        rules:
+          - action: Accept
+            cidr: 10.128.0.0/14      # pod
+            peerCIDR: 10.128.0.0/14  # pod
+            sampling: 50
+          - action: Accept
+            cidr: 10.128.0.0/14      # pod
+            peerCIDR: 172.30.0.0/16  # service
+            sampling: 50
+          - action: Accept
+            cidr: 0.0.0.0/0
+            sampling: 1
+```
+
+<p style="text-align: center">Listing 5: Flow filter with different sampling value</p>
+
+The last rule with CIDR 0.0.0.0/0 is necessary because without it, it will only process the internal traffic packets.  If sampling is not specified, it uses the sampling value from FlowCollector.
+
+#### Include packet drops
+
+Another new option is **pktDrops**.  With **pktDrops: true** and **action: Accept**, it includes the packet only if it's dropped.  The prerequisite is that the eBPF feature, **PacketDrop** is enabled, which requires eBFP to be in **privileged** mode.  Note this currently is not supported if you enable the NetworkEvent feature.  Listing 6 shows an example configuration.
+
+```
+spec:
+  agent:
+    type: eBPF
+    ebpf:
+      privileged: true
+      features:
+        - PacketDrop
+      flowFilter:
+        enable: true
+        rules:
+          - action: Accept
+            cidr: 10.1.0.0/16
+            sampling: 1
+            pktDrops: true
+```
+
+<p style="text-align: center">Listing 6: Flow filter with packet drop example</p>
+
+For more information and use cases on eBPF Flow Filter, see the article on [Enhancing NetObserv By Introducing Multi Rules Flow filtering capability in eBPF](https://netobserv.io/posts/enhancing-netobserv-by-introducing-multi-rules-flow-filtering-capability-in-ebpf/).
+
+
+### UDN Observability
+
+Kubernetes networking consists of a flat Layer 3 network and a single IP address space where every pod can communicate with any other pod.  In a number of use cases, this is undesirable.  OVN-Kubernetes provides another model called [User-Defined Networks](https://docs.openshift.com/container-platform/4.18/networking/multiple_networks/primary_networks/about-user-defined-networks.html) (UDN).  It supports microsegmentation where each network segment, which could be Layer 2 or Layer 3, is isolated from one another.
```

Comment:
Maybe add a sentence at the end stating that 1.8 add observability support for UDNs including topology, etc.